PENDING.md

```diff
@@ -16,11 +16,16 @@ BREAKING CHANGES
 FEATURES
 
 * Gaia REST API (`gaiacli advanced rest-server`)
+    * [gaia-lite] [\#2182] Added LCD endpoint for querying redelegations
```

Comment:
This should have been in breaking changes cc: @sunnya97 @jackzampolin. I'll update it